lib/dialogs.dart

```diff
@@ -91,7 +168,12 @@ class SharingDialog extends DDialog {
     _closeButton = new DButton.button(text: 'Close', classes: 'default');
     _closeButton.onClick.listen((_) => hide());
     _div = new DElement.tag('div')..layoutVertical();
-
+    _exportButton = new DButton.button(text: 'Export', classes: 'default');
```

Comment:
Instead of `Export`, what about `Create embeddable link` or `View embeddings options`?